lib/cloudwatch-integration.js

```diff
@@ -113,6 +122,13 @@ function retrySubmit(aws, payload, times, cb) {
 }
 
 lib.getToken = function(aws, groupName, streamName, retentionInDays, cb) {
+  var existingNextToken = lib.getToken.lastNextToken[previousKeyMapKey(groupName, streamName)];
+  if(existingNextToken !== null && existingNextToken !== undefined) {
```

Comment:
I would call it `nextToken` as we're not sure it's there yet, also what about simplifying it to 

```JS
if (nextToken != null) {
```

? `!=` checks for both `null` and `undefined`.

Reply:
Lol I don't know why I didn't just do that. Will go through and make the other suggested changes today. Cheers :)